### ISSUE TITLE
moving panes new 8.0.1

### ISSUE THREAD

**Issue by andrewufrank** [leksah/leksah#299]
i have a clean installation and can split windows but cannot move files between panes. it is important that files shown in one pane can be moved to another pane to have the "right" files visible at the same time.

what is wrong? (setup? default values?)
thank you!

**Comment by jaccokrijnen** [comment]
I just noticed this too, on which OS are you, and which version of GTK do you have? I'm on Ubuntu 15.10 with GTK 3.16.

**Comment by jaccokrijnen** [comment]
As temporary workaround, you can use View -> Move Left/Right/Up/Down and their shortcuts to move panes.

**Comment by andrewufrank** [comment]
i just upgraded (linux debian stretch (testing) ) with gtk 0.14.3 (the haskell package) resp 3.20.5 for libgtk3. it is hard to see which package did the trick.

**Comment by jaccokrijnen** [comment]
This still remains a problem on 3.16

**Comment by achirkin** [comment]
I have the same problem.
Just installed latest leksah from source, following the readme.
Using Ubuntu 16.04, GNOME Shell 3.18.5.
I am not sure how much is the following related, but I constantly get these two messages in console in random order:

```
(leksah:18501): Gtk-CRITICAL **: gtk_tree_model_get_iter: assertion 'path->depth > 0' failed
(leksah:18501): Gtk-CRITICAL **: gtk_notebook_get_tab_label: assertion 'list != NULL' failed
```

**Comment by andrewufrank** [comment]
i just did a clean compile of 16.1 and the problem with the moving of panes still persists (the alt-shif-arrow-left works).

**Comment by andrewufrank** [comment]
including here (but perhaps a separate issue): when opening a file from the workspace pane it should open not in the workspace pane (i have set to toolCategory), but one of the other explorerer category windows.

**Comment by andrewufrank** [comment]
fixed in 0.16.3.1 - thank you!
